tools/templates/profile.mustache

```diff
@@ -55,27 +51,12 @@
           </a>
         </div>
       </li>
+
       <li>
-        Exchange the <strong>authorization code</strong> for an
```

Comment:
I think this information and the link to DOC is useful and should be brought back, but it should be mentioned that the back-end is handling this for you.

Reply:
Done